UEVR/MainWindow.xaml.cs

```diff
@@ -1068,6 +1070,20 @@ public partial class MainWindow : Window {
         private static extern bool IsWow64Process([In] IntPtr hProcess, [Out] out bool wow64Process);
 
         private bool IsInjectableProcess(Process process) {
+
+            List<string> excludedProcesses = new List<string>();
+
+            // Check if the excluded processes file exists
+            if (File.Exists(excludedProcessesFile)) {
+                // Read excluded process names from the text file
+                excludedProcesses = File.ReadAllLines(excludedProcessesFile).ToList();
+            }
+        
+            // Check if the process name is in the excluded list
+            if (excludedProcesses.Contains(process.ProcessName)) {
+                return false;
+            }
+            
```

Comment:
Put all of this inside the try-catch block below

Reply:
Done! Also put all the logic on the if block for better encapsulation